LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DeviceLoaderSynchronizer.cs

```diff
@@ -146,60 +118,70 @@ private async Task Load()
             }
         }
 
-        private async Task<List<LoRaDevice>> CreateDevicesAsync(IReadOnlyList<IoTHubDeviceInfo> devices)
+        protected async Task CreateDevicesAsync(IReadOnlyList<IoTHubDeviceInfo> devices)
```

Comment:
Question what would happen if

We have device A and B sharing the same devaddress, A is loaded on gateway 1. Now B starts sending messages and Gateway 2 wins the race, now the two gateways have both device in cache

Reply:
If I understood correctly the connection is opened only at message send so this should not be a problem, but is the twin get not potentially conflicting?